content/en/continuous_integration/tests/python.md

```diff
@@ -27,6 +27,7 @@ Supported Python interpreters:
 Supported test frameworks:
 * pytest >= 3.0.0
   * pytest < 5 when using Python 2
+* pytest-benchmark >= 3.1.0
```

Comment:
we want to add here the version of `dd-trace-py` from which we support this framework (to be determined, because the release hasn't happened yet).

Also, if there's any restriction with python versions we want to add them here too

Reply:
Good catch! Just added the Python version restriction and will wait for a release.